libs/librrgraph/src/base/rr_graph_view.h

```diff
@@ -584,14 +586,23 @@ class RRGraphView {
      * @example
      * RRGraphView rr_graph; // A dummy rr_graph for a short example
      * RRNodeId node; // A dummy node for a short example
-     * for (RREdgeId edge : rr_graph.edges(node)) {
+     * for (t_edge_size edge : rr_graph.edges(node)) {
      *     // Do something with the edge
      * }
      */
     inline edge_idx_range edges(const RRNodeId& id) const {
```

Comment:
I think the comments above this function don’t clearly explain that the returned ID is not the RREdgeId, but rather the index among the node’s outgoing edges. If you could update the comments to make that distinction clearer, and also show one of the functions that uses this index instead of RREdgeId, that would be great.